help/destinations/catalog/ecommerce/sap-commerce.md

```diff
@@ -0,0 +1,329 @@
+![Image from Platform UI showing an example of customerNumber identity mapping.](../../assets/catalog/ecommerce/sap-commerce/mapping-identities.png)
+
+#### Mapping attributes
+
+To add any other attributes you want to update between your XDM profile schema and your [!DNL SAP Subscription Billing] account repeat the steps below:
+1. In the **[!UICONTROL Mapping]** step, select **[!UICONTROL Add new mapping]**. You can now see a new mapping row on the screen.
+![Platform UI screenshot with add new mapping button highlighted.](../../assets/catalog/ecommerce/sap-commerce/mapping-add-new-mapping.png)
+1. In the **[!UICONTROL Select source field]** window, choose the **[!UICONTROL Select attributes]** category and select the XDM attribute.
+![Platform UI screenshot selecting Last Name as a source attribute.](../../assets/catalog/ecommerce/sap-commerce/mapping-select-source-attribute.png)
+1. In the **[!UICONTROL Select target field]** window, choose **[!UICONTROL Select custom attributes]** category and type the name of the [!DNL SAP Subscription Billing] attribute from the list of customer [schema](https://api.sap.com/api/BusinessPartner_APIs/schema) attributes.
+![Platform UI screenshot where lastName is defined as the target attribute.](../../assets/catalog/ecommerce/sap-commerce/mapping-select-target-attribute.png)
+
+>[!IMPORTANT]
+>
+> Target field names are case sensitive and should match the [!DNL SAP Subscription Billing] attribute names. The only exception for this is `country` where you should use `countryCode` instead. [!DNL SAP Subscription Billing] supports alpha-2 (ISO 3166) country codes. The value is case sensitive and must be between 0-3 characters, hence ensure you provide exactly as defined else you would encounter errors: `The country code {} does not exist` or `size must be between 0 and 3`.
+
+#### Map `mandatory` attributes for the selected customer type
+
+Mandatory attribute mappings are dependent on the **[!UICONTROL Type of Customer]** you had selected. To map the mandatory attributes select from the below:
+
+>[!BEGINTABS]
+
+>[!TAB Individual customer]
+
+| Source Field | Target Field | Mandatory |
+| --- | --- | --- |
+| `xdm: person.lastName` | `Attribute: lastName` | Yes |
+| `xdm: workAddress.countryCode` | `Attribute: countryCode` | Yes |
+
+>[!TAB Corporate customer]
+
+| Source Field | Target Field | Mandatory |
+| --- | --- | --- |
+| `xdm: b2b.companyName` | `Attribute: company` | Yes |
+| `xdm: workAddress.countryCode` | `Attribute: countryCode` | Yes |
+
+>[!ENDTABS]
+
+#### Mapping additional attributes
+
+You can then add any additional mappings between your XDM profile schema and the [!DNL SAP Subscription Billing] [schema](https://api.sap.com/api/BusinessPartner_APIs/schema) attributes for a customer as shown below:
+
+>[!BEGINTABS]
+
+>[!TAB Individual customer]
+
+| Source Field | Target Field | Mandatory |
+| --- | --- | --- |
+| `xdm: person.name.firstName` | `Attribute: firstName` |
```

Comment:
table: if these updates are not mandatory, let's please add "No" to the third column